packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/marker.dart

```diff
@@ -337,7 +367,7 @@ class Marker implements MapsObject<Marker> {
     return 'Marker{markerId: $markerId, alpha: $alpha, anchor: $anchor, '
         'consumeTapEvents: $consumeTapEvents, draggable: $draggable, flat: $flat, '
         'icon: $icon, infoWindow: $infoWindow, position: $position, rotation: $rotation, '
-        'visible: $visible, zIndex: $zIndex, onTap: $onTap, onDragStart: $onDragStart, '
+        'visible: $visible, zIndex: $zIndex, zIndexInt:$zIndexInt onTap: $onTap, onDragStart: $onDragStart, '
```

Comment:
Printing `zInedxInt` doesn't add any information since the same information is already available from `zIndex`, so this can be reverted.